gluten-core/src/main/scala/io/glutenproject/expression/LambdaFunctionTransformer.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.glutenproject.expression
+
+import io.glutenproject.substrait.expression.{ExpressionBuilder, ExpressionNode}
+
+import org.apache.spark.internal.Logging
+import org.apache.spark.sql.catalyst.expressions.LambdaFunction
+
+import java.util.ArrayList
+
+class LambdaFunctionTransformer(
+    substraitExprName: String,
+    function: ExpressionTransformer,
+    arguments: Seq[ExpressionTransformer],
+    hidden: Boolean = false,
+    original: LambdaFunction)
+  extends ExpressionTransformer
+  with Logging {
+
+  override def doTransform(args: Object): ExpressionNode = {
+    // hidden not used as not supported in Velox
```

Comment:
This comment looks confusing to me, can we explain its meaning?

Reply:
@yma11 What does this comment mean, could you help explain a bit?

Reply:
I should have updated it but seems change not committed. Let me update.

Reply:
done.

Reply:
Do we need to fallback this function when `hidden == true` to avoid possible failure?